frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -43,6 +46,22 @@ const handleError = (res: any, flags: string[], dispatch: Dispatch, cb: FeatureD
   }
 };
 
+export const detectRGW: FeatureDetector = async (dispatch) => {
+  let id = null;
+  const logicHandler = () =>
+    k8sList(StorageClassModel).then((data) => {
+      const provisioners = data.map((sc) => sc.provisioner);
+      if (provisioners.includes('openshift-storage.ceph.rook.io/bucket')) {
+        dispatch(setFlag(RGW_FLAG, true));
+        clearInterval(id);
+      } else {
+        dispatch(setFlag(RGW_FLAG, false));
+        clearInterval(id);
+      }
+    });
+  id = setInterval(logicHandler, 10000);
```

Comment:
nit: create variable for SECOND
```suggestion
  id = setInterval(logicHandler, 10* SECOND);
```